my_pyscf/pdmet/_pdmet.py

```diff
@@ -0,0 +1,436 @@
+from pyscf import lo, lib, ao2mo
+from pyscf.pbc import gto, scf
+from functools import reduce
+import numpy as np
+from mrh.my_pyscf.pdmet import basistransformation as bt
+from mrh.my_pyscf.pdmet.localization import Localization
+from mrh.my_pyscf.pdmet.fragmentation import Fragmentation
+
+# Author: Bhavnesh Jangid <jangidbhavnesh@uchicago.edu>
+
+get_basis_transform = bt.BasisTransform._get_basis_transformed
+
+class _pDMET:
```

Comment:
Can you not define this as a child class of `dmet._dmet._DMET` and save many lines of code?